.github/workflows/python.yml

```diff
@@ -32,8 +32,7 @@ jobs:
         sudo apt update
 
     - uses: Chocobo1/setup-ccache-action@v1
-      # the ccache version on ubuntu-18.04 is too old
-      if: matrix.os != 'ubuntu-18.04'
+      if: runner.os != 'Windows'
```

Comment:
I was under the impression that your ccache action would just be a no-op on windows, and that it's not necessary to have a condition to only run it on non-windows.

I would have expected this condition to just be removed instead.

Reply:
the action outputs a warning about being a no-op on windows. This gets surfaced in the "PR checks" UI like an error on the workflow. See https://github.com/Chocobo1/setup-ccache-action/issues/2

Reply:
> I was under the impression that your ccache action would just be a no-op on windows, and that it's not necessary to have a condition to only run it on non-windows.
> 
> I would have expected this condition to just be removed instead.

Personally I would like to have a conditional, but ultimately it is up to you whether you are ok with a warning...